README.md

```diff
@@ -47,10 +47,10 @@ you should run the tests:
 pip install -e .
 
 # Run the python tests. This should not give you a few sucessful example tests
+# You may need to install nbval first: pip install nbval
```

Comment:
Maybe we should instead say `pip install -e ".[test, examples]"` in the above line?

Reply:
Indeed, it would be better. I will update the PR